ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/mapper/PaymentIntentionMapper.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.cmc.ccd.domain.mapper;
```

Comment:
please move this also top mapper package

Reply:
please check my last commit for this PR.